packages/lisk-cryptography/.eslintrc.json

```diff
@@ -2,5 +2,8 @@
 	"extends": ["../../.eslintrc.json"],
 	"globals": {
 		"naclInstance": true
+	},
+	"rules": {
+		"import/no-cycle": "off"
```

Comment:
😭

Reply:
😯 I'll create a follow-up issue and add it to 1.1.0.

Reply:
https://github.com/LiskHQ/lisk-elements/issues/775